site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -104,12 +107,17 @@ An extension can add or remove rules dynamically using the [updateDynamicRules][
 An extension can update the set of enabled static rulesets using the [updateEnabledRulesets][14] API
 method.
 
+- The number of static rulesets which are enabled at one time must not exceed
+  [MAX_NUMBER_OF_ENABLED_STATIC_RULESETS][18].
 - The number of rules across enabled static rulesets across all extensions must not exceed the
   [global limit][15]. Calling [getAvailableStaticRuleCount][10] is recommended to check the number
   of rules an extension can still enable before the global limit is reached.
 - The set of enabled static rulesets is persisted across sessions but not across extension updates.
   The `rule_resources` manifest key will determine the set of enabled static rulesets on initial
   extension install and on each subsequent extension update.
+- Errors and warnings about invalid static rules are only displayed for unpacked extensions. Invalid
+  static rules in packed extensions are ignored. It's therefore important to verify your static
+  rulesets are valid by testing an unpacked version of your extension.
```

Comment:
I'm not quite happy about how these points apply to both manifest enabled and `updateEnabledRulesets` enabled static rulesets. I worry that their position makes it seem like they only apply to the latter. I can't see how better to do it though, what do you think?

Reply:
That's a good point. How about moving this to the Rule Resources section above since that deals with static rulesets.

Reply:
OK makes sense, will do. I'll leave the other points here though, since they are already covered in the Rule Resources section.